modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,72 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+== Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily in order to clean up any stranded IP allocations that may result in exhausting IPs and hence prevent new Pods from getting an IP allocated to them.
```

Comment:
```suggestion
The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that may result in exhausting IPs and therefore prevent new Pods from getting an IP allocated to them.
```
Vale is suggesting 'therefore' instead of 'hence' but I can't find where that lives in the style guidance. Also, the 'exhausting IPs' makes the sentence a little confusing, maybe rephrase to be more clear?